examples/angular-cli/src/stories/addons/READEME.stories.mdx

```diff
@@ -0,0 +1,7 @@
+import { Meta } from '@storybook/addon-docs/blocks';
```

Comment:
There is a typo in the file name: `READEME` -> `README`

Reply:
🙄  🙇  :)